GVFS/GVFS.FunctionalTests/Tests/GitCommands/UpdateIndexTests.cs

```diff
@@ -63,6 +63,16 @@ public void UpdateIndexRemoveAddFileOpenForWrite()
             GitHelpers.InvokeGitAgainstGVFSRepo(this.Enlistment.RepoRoot, "update-index --add Test_ConflictTests/AddedFiles/AddedByBothDifferentContent.txt");
         }
 
+        [TestCase]
+        [Category(Categories.MacTODO.M4)]

```

Comment:
I have not tried running this on the Mac - I will test and see what happens (or remove this, if possible)

Reply:
This test fails on the "cleanup" code (not related to the protocol changes). After "resetting" the repository, there are unexpected changes in the test repository. Because of this (and because the failure is not related to the protocol changes), I will leave this as a to for Mac